app/controllers/templates_controller.rb

```diff
@@ -59,4 +66,10 @@ def destroy
   def template_params
     params.require(:template).permit(:name, :question_ids => [])
   end
+
+  def check_admin
```

Comment:
Why did you add this back? Templates is something that interviewer should manage? This already was removed in one of previous commits.